### DISCUSSION TITLE
Straigten/Indent Lines by Word

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#457] [Q&A] [answered]
Hi,

I'd like to go from this:

```
15 Hello
30 Hello Foo
1.0 Bar
30 Bar Foo
3.15 Berry
2.0 Zah
```

To this:

```
15    Hello
30    Hello Foo
1.0   Bar
30    Bar Foo
3.15  Berry
2.0   Zah
```

How can I achieve the latter formatting with meow and beacon mode?

**Comment by DogLooksGood** [comment, marked as answer]
You don't need meow for this, it's easy. Select these lines, `M-x align-regexp`, then provide a single space.

**Reply by jgarte** [reply]
Thanks!
